evm-e2e/test/utils.ts

```
@@ -81,3 +77,7 @@ export const deployContractNibiruOracleChainLinkLike = async (): Promise<{
   await contract.waitForDeployment();
   return { oraclePair, contract };
 };
+
+export const numberToHex = (num: Number) => {
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)